src/modules/Home/components/Home/Home.tsx

```diff
@@ -0,0 +1,347 @@
+import cn from 'classnames';
```

Comment:
split this component on subcomponents. having over 300 rows in one file is not cool. Make small components instead

Reply:
done